### DISCUSSION TITLE
soup doesn't upgrade to 2.4 from 2.3.270

### DISCUSSION THREAD

**Discussion by harleytomlinsonOMB** [Security-Onion-Solutions/securityonion#11634] [General] [answered]
Distributed install (new, building it out still)
Internet access 
VM with 4 cores and 16384MB RAM 200GB disk

Soup output states you are already running the latest version of Security Onion
/etc/soversion states 2.3.270

How do I upgrade to 2.4? nuke and pave? No in place upgrade possible?

**Comment by Jaap79** [comment]
There is no in-place upgrade at this time..

**Reply by Jaap79** [reply]
I stand corrected; there is an option to upgrade in place apparantly (check @TOoSmOotH  's note here) but it seems quite experimental at this time.

**Reply by TOoSmOotH** [reply]
We have had several customers upgrade successfully but if you really don't need to retain the data it's worth starting fresh.

**Comment by TOoSmOotH** [comment, marked as answer]
https://docs.securityonion.net/en/2.4/appendix.html

**Reply by harleytomlinsonOMB** [reply]
Thanks!
